File: Iniciante/URI 1011.cs
using System; 

class URI {

    public static void Main(string[] args) { 

        double pi, volume;
        short R;
        
        R= Int16.Parse(Console.ReadLine());
        pi= 3.14159;
        
        volume= (4/3.0) * pi * Math.Pow(R, 3);
        
        Console.WriteLine("VOLUME = {0:F3}", volume);
    }
}

File: Iniciante/URI 1011.cpp
#include <stdio.h>
#include <math.h>

int main()
{

  double pi, volume;
  unsigned short int R;

  scanf("%hd", &R);
  pi = 3.14159;

  volume = (4 / 3.0) * pi * pow(R, 3);

  printf("VOLUME = %.3lf", volume);
  printf("\n");

  return 0;
}